backend/server.js

```diff
@@ -0,0 +1,247 @@
+var express = require('express'); 
+var app = express(); 
+  
+const bodyParser = require('body-parser'); 
+var server = app.listen(3000); 
+  
+app.use(bodyParser.json()); 
+app.use(bodyParser.urlencoded({ extended: true })); 
+  
+app.post('/postdata', (req, res) => { 
+	console.log(req.body); 
+   var data = req.body.data; // your data 
+	console.log(data); 
+
+   var user_id = req.body.user_id;
+   console.log(user_id); 
+
+   var password = req.body.password;
+   console.log(password); 
+
+   // do something with that data (write to a DB, for instance) 
```

Comment:
Codacy has a fix for the issue: [Replace `·//·do·something·with·that·data·(write·to·a·DB,·for·instance)·` with `//·do·something·with·that·data·(write·to·a·DB,·for·instance)`](https://app.codacy.com/gh/CPEN321HELP/CPEN321/pullRequest?prid=9852520)
```suggestion
  // do something with that data (write to a DB, for instance)
```